deployment/7_upgrade_ens.js

```diff
@@ -0,0 +1,31 @@
+const TestENSRegistry = require('../build/TestENSRegistry');
+const TestReverseRegistrar = require('../build/TestReverseRegistrar');
+const ENSResolver = require('../build/ENSResolver');
+const ENSManager = require('../build/ENSManager');
+
+const deploy = async (network) => {
+    const config = configurator.config;
```

Comment:
We need to create the `configurator`, the `deployer` and the `abiUploader` first using:
```
    const manager = new DeployManager(network);
    await manager.setup();
    const configurator = manager.configurator;
    const deployer = manager.deployer;
    const abiUploader = manager.abiUploader;
```